### PR TITLE
Travis ewoms

### PR DESCRIPTION
**Pull request by andlaus**
this hopefully adds the  eWoms module to the travis CI system. it is basically the union of #164 and #163. Thanks to @joakim-hove for the initial patch.

### CODE REVIEW

File: cmake/Modules/OpmSatellites.cmake [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # - Build satellites that are dependent of main library
+
+set(ADD_DISABLED_CTESTS YES CACHE BOOL "Add the tests which are disabled due to failed preconditions to the ctest output (this makes ctest return an error if such a test is present)")
```
Comment (akva2) [inline cmake/Modules/OpmSatellites.cmake]: use
```cmake
option(ADD_DISABLED_CTESTS "Add .." ON)
```

Comment (andlaus) [inline cmake/Modules/OpmSatellites.cmake]: cool: I was wondering how to do this properly. fixed, thanks!

File: cmake/Modules/OpmSatellites.cmake [hunk 2]
```diff
@@ -1,4 +1,8 @@
 # - Build satellites that are dependent of main library
+
+option(ADD_DISABLED_CTESTS "Add the tests which are disabled due to failed preconditions to the ctest output (this makes ctest return an error if such a test is present)" ON)
+mark_as_advanced(ADD_DISABLED_CTESTS)
```
Comment (akva2) [inline cmake/Modules/OpmSatellites.cmake]: we do want this shown in the gui imo.

Comment (andlaus) [inline cmake/Modules/OpmSatellites.cmake]: you mean making it non-advanced? I think it is a pretty esoteric feature, so it should be hidden by default. That said, if other people want this to be shown by default, I'll budge...

Comment (akva2) [inline cmake/Modules/OpmSatellites.cmake]: nothing i care too much about, but i thought it was a left-over and not a conscious choice. we'll leave it as-is.

Comment (andlaus) [inline cmake/Modules/OpmSatellites.cmake]: same feeling here: I've no really strong feelings about this, I just think that in this regard "less is more", i.e. only the options which most users are likely to care about should be displayed by default.

**Comment by akva2** [comment]
jenkins build this with downstreams please

**Comment by akva2** [comment]
jenkins is happy, travis is happy, andreas is happy, i am happy. and i think @joakim-hove should be happy about his code, and the rest falls under my maintenance. i'll merge. thanks all.

**Comment by joakim-hove** [comment]
> and i think @joakim-hove should be happy 

Sure :grinning:

**Pull request merged**